def print_error(lineno,message="error"):
    print("%s at line %d" % (message,lineno))
print_error(42)
